test/conformance/source/environment.cpp

```diff
@@ -199,6 +199,8 @@ DevicesEnvironment::DevicesEnvironment(int argc, char **argv)
         error = "Could not find any devices associated with the platform";
         return;
     }
+    // TODO: Add a parameter to make the test work on one device only
+    count=1;
```

Comment:
Devcloud has 4 graphics cards. CTS tests are not designed to run on a multi-device platform. For now, I have hardcoded (variable count) that the tests run on the first device. It is now possible to focus on the tests that are failing (some tests in the matchfile have passed and new ones have failed) in these CTS binary:
 12 - memory-adapter_level_zero (Failed)
16 - sampler-adapter_level_zero (Failed)
The next step would be to:
- add a parameter to gtest so that the tests would only be run on one device 
 - add PCI address in GetPlatformAndDeviceName function to make it possible to run on a multi-device platform. 
(I have a solution to make it possible)

Reply:
> * add a parameter to gtest so that the tests would only be run on one device
> * add PCI address in GetPlatformAndDeviceName function to make it possible to run on a multi-device platform. (I have an idea to make it possible) In addition, it will be necessary to change matchfiles.

1. Yes, we can't just hardcode it like this. Please make an option as part of this PR.
2. Please create a separate issue for this.

Reply:
I have done an option for this